source/support.tex

```diff
@@ -3378,7 +3378,7 @@
 \tcode{parmN}.}
 If the parameter
 \tcode{parmN}
-is of a reference type, or of a type that is not compatible with the
+is of a reference type, or of a type that is not the same as the
```

Comment:
To be clear, the signed/unsigned relaxation is a separate rule that's applied to arguments that don't match declared parameters (either because there's no prototype, or because the prototype ends in an ellipsis), and is not part of the notion of "compatible".